multiversion/src/main/scala/multiversion/outputs/ArtifactOutput.scala

```diff
@@ -25,6 +25,7 @@ final case class ArtifactOutput(
       s"_${dependency.publication.classifier.value}"
     else ""
   val label: String = dependency.bazelLabel
+  val sourcesLabel: String = label + "_sources"
```

Comment:
```suggestion
  private val baseLabel = dependency.bazelLabel
  val label: String = baseLabel + "_jar"
  val sourcesLabel: String = baseLabel + "_sources"
```

Since the `http_file` target is not visible, we could have a suffix for it as well to make sure that we never run into name conflicts between some jar and another source jar (though having a jar whose version ends in `_sources` seems very unlikely).

Reply:
Yea. Sounds good.

Reply:
Actually too much work to update all the tests

```diff
=> Diff (- obtained, + expected)
   "  \"//foo:foo\" -> \"@maven//:reflections\"",
-  "  \"@maven//:_aopalliance_aopalliance_1.0_jar\"",
-  "  \"@maven//:_aopalliance_aopalliance_1.0_jar\" -> \"@maven//:aopalliance/aopalliance/1.0/aopalliance-1.0.jar\"",
-  "  \"@maven//:_com.google.guava_guava_16.0.1_jar\"",
-  "  \"@maven//:_com.google.guava_guava_16.0.1_jar\" -> \"@maven//:com.google.guava/guava/16.0.1/guava-16.0.1.jar\"",
-  "  \"@maven//:_com.google.guava_guava_20.0_jar\"",
-  "  \"@maven//:_com.google.guava_guava_20.0_jar\" -> \"@maven//:com.google.guava/guava/20.0/guava-20.0.jar\"",
-  "  \"@maven//:_javax.inject_javax.inject_1_jar\"",
-  "  \"@maven//:_javax.inject_javax.inject_1_jar\" -> \"@maven//:javax.inject/javax.inject/1/javax.inject-1.jar\"",
-  "  \"@maven//:_org.javassist_javassist_3.21.0-GA_jar\"",
-  "  \"@maven//:_org.javassist_javassist_3.21.0-GA_jar\" -> \"@maven//:org.javassist/javassist/3.21.0-GA/javassist-3.21.0-GA.jar\"",
+  "  \"@maven//:_aopalliance_aopalliance_1.0\"",
+  "  \"@maven//:_aopalliance_aopalliance_1.0\" -> \"@maven//:aopalliance/aopalliance/1.0/aopalliance-1.0.jar\"",
+  "  \"@maven//:_com.google.guava_guava_16.0.1\"",
+  "  \"@maven//:_com.google.guava_guava_16.0.1\" -> \"@maven//:com.google.guava/guava/16.0.1/guava-16.0.1.jar\"",
+  "  \"@maven//:_com.google.guava_guava_20.0\"",
+  "  \"@maven//:_com.google.guava_guava_20.0\" -> \"@maven//:com.google.guava/guava/20.0/guava-20.0.jar\"",
+  "  \"@maven//:_javax.inject_javax.inject_1\"",
+  "  \"@maven//:_javax.inject_javax.inject_1\" -> \"@maven//:javax.inject/javax.inject/1/javax.inject-1.jar\"",
+  "  \"@maven//:_org.javassist_javassist_3.21.0-GA\"",
+  "  \"@maven//:_org.javassist_javassist_3.21.0-GA\" -> \"@maven//:org.javassist/javassist/3.21.0-GA/javassist-3.21.0-GA.jar\"",
```

so I revert this change.